common/config/azure-pipelines/jobs/version-bump.yaml

```diff
@@ -187,7 +187,40 @@ jobs:
         displayName: Get Branch Name
 
       - bash: git checkout -b $(releaseBranchName)
-        displayName: Create and publish branch
+        displayName: Create release branch and checkout
+
+      - bash: |
+          docsYamlPath="common/config/azure-pipelines/templates/gather-docs.yaml"
+          releaseBranch=$(releaseBranchName)
+
+          # delimiter
+          IFS="."
+          read -ra splitArray <<< "$releaseBranch"
+
+          currentMinorVersion=${splitArray[1]}
+          previousMinorVersion=$((currentMinorVersion - 1))
+
+          # check if this release is major or minor version release
+          # if minor version bump, use last recent minor version
+          # if major version bump, `gather-docs.yaml` needs to be edited
+          # manually to be the most recent previous major release
+          # e.g. if this release is `release/5.0.x`, value in `gather-docs.yaml`
+          # should be `release/4.<whatever_last_minor_release_version_was>.x`
+          if [ $((previousMinorVersion)) -lt 0 ]
+          then
+            echo "This is is a major release. Edit \"gather-docs.yaml\" manually."
+          else
+            previousReleaseBranch="${releaseBranch/"$currentMinorVersion.x"/"$previousMinorVersion.x"}"
+            escapedReleaseBranchName="${previousReleaseBranch//\//\\\/}"
+            search="branchName: refs\/heads\/master"
+            replace="branchName: refs\/heads\/"$escapedReleaseBranchName""
+            echo "$search"
+            echo "$replace"
+            sed -i "s/$search/$replace/g" "$docsYamlPath"
```

Comment:
do we need to replace every occurrence of refs/heads/master with that `/g` option? There should only be one spot right? I'm thinking in case of side effects

Reply:
yeah, gotcha, do you think this warrants a comment or a disclaimer? Or maybe this comment thread is enough

Reply:
Ya I can add a comment, no problem. Thanks